osp/osp.h

```diff
@@ -61,6 +61,7 @@ typedef enum
 typedef enum
 {
   OSP_SCAN_STATUS_ERROR = -1, /**< Error status. */
+  OSP_SCAN_STATUS_PENDING,    /**< Init pending. */
```

Comment:
Don't know if anything stores these, but it's safer to add to the end.

For example if gvmd was storing these states as integers in the db, then the stored states would all be wrong.